### ISSUE TITLE
Remove support for Python 2.6

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8017]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8017|
|Type|enhancement|
|Created|2015-09-07 05:53:01Z|
|Branch|https://github.com/twisted/twisted/tree/remove-twosix-8017|

<details><summary>Searchable metadata</summary>

```
trac-id__8017 8017
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__EndOfLifePython2_6 EndOfLifePython2.6
branch__branches_remove_twosix_8017 branches/remove-twosix-8017
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1441605181378537 1441605181378537
changetime__1441780592780531 1441780592780531
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [45656]) Branching to remove-twosix-8017.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Tests are running, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Looks good. Thanks!

There are a few errors:

```
************* Module twisted.test.test_twisted
C0301:289,0: Line too long (99/79)
W9501:289,12:RequirementsTests.test_oldPythonPy3: String formatting operations should always use a tuple for non-mapping values

```

Can the news fragment contain a link to the ML discussion ?

Please fix and merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45660]) Merge remove-twosix-8017: Remove Python 2.6 support.

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8017](https://github.com/twisted/twisted/issues/8017)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

I decided a link to the ML discussion wasn't really required -- it's as all the previous Python desupportings.
